src/web/mjs/connectors/ComicK.mjs

```diff
@@ -40,7 +40,7 @@ export default class ComicK extends Connector {
 
     async _getMangasFromPage(page) {
         try {
-            const uri = new URL('/v1.0/search?page=' + page, this.apiurl);
+            const uri = new URL('/v1.0/search?limit=49&page=' + page, this.apiurl); //if we keep the limit to 49 we can fetch all 25 pages
```

Comment:
This comment is not appropriate for source code, changes can be traced by developer in commit history to the the reason this was changed